next/backend/graphql/queries/InbaArticles.graphql

```diff
@@ -68,6 +68,15 @@ fragment InbaArticleEntity on InbaArticleEntity {
       }
     }
     content
+    inbaRelease {
+      ...InbaReleaseEntityResponse
+    }
+  }
+}
+
+fragment InbaReleaseEntityResponse on InbaReleaseEntityResponse {
+  data {
+    ...InbaReleaseEntity
```

Comment:
First just coding style, we agreed in team, that for consistency, we will create fragments on Entities. Moreover, there is no need for fragment `InbaReleaseEntityResponse`. I would just write 
```
...
 content
    inbaRelease {
      data {
        ...InbaReleaseEntity
      }
    }
  }
```

Second, more important, Graphql's main advantage is that you can specify only fields you need, so you always know what you get, and requests are smaller.

So for this case, I wouldn't use `InbaReleaseEntity` fragment, but I would reuqest only fields we need (title, date, url, ...?)
You could specify new fragment on `InbaReleaseEntity`, name it for example `InbaArticleReleaseEntity` (this is always a challange to find the right name), but you don't have to specify a fragment at all, because this part is not reused anywhere else.

So I would just write 
```
... 
content
inbaRelease {
  data {
    attributes {
      fields_you_need
    }
  }
}
```